externs/browser/html5.js

```diff
@@ -3040,8 +3040,7 @@ WebSocket.prototype.onopen;
 
 /**
  * An event handler called on message event.
- * TODO(blickly): Change this to MessageEvent<*> and add casts as needed
- * @type {?function(!MessageEvent<?>): void}
+ * @type {?function(!MessageEvent<string|ArrayBuffer|Blob>): void}
```

Comment:
I don't think the spec allows `null`, so the `MessageEvent` type should be `!MessageEvent<string|!ArrayBuffer|!Blob>`.
Is that right?

Reply:
That does seem right. I will push the change. Would you prefer if I rebased and squashed the commit?

Reply:
Thanks for asking, but I don't need the rebase/squash.
My personal workflow for importing PRs will actually work better if you don't do that.